### PR TITLE
mob limit

### PR DESCRIPTION
**Pull request by Bod9001**
why because the thing that always is responsible for tanking performances bloody mobs, 
so adds a limit to the number of mobs that can be active , can change limit is needed
Looking at the profiles there was 81 hostile mobs looking for players at the same time

### CODE REVIEW

**Comment by corp-0** [comment]
the codacy warnings make me uneasy with the static solution, not going to lie.

**Comment by Bod9001** [comment]
it is a static cap Though, applying across all classes so it makes sense to be static

File: UnityProject/Assets/Scripts/NPC/AI/MobController.cs [hunk 1]
```diff
@@ -3,30 +3,56 @@
...
 		}
 
 		private void OnEnable()
 		{
 			if (CustomNetworkManager.IsServer == false) return;
-		
+			SceneManager.activeSceneChanged += OnRoundRestart;
+			if (CurrentMobs > 20)
+			{
+				return;
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/NPC/AI/MobController.cs:40]: Could we send this to admin chat? Just to keep them informed

Comment (Bod9001) [reply UnityProject/Assets/Scripts/NPC/AI/MobController.cs:40]: ok cool will do

**Review by corp-0 — APPROVED** [review]

**Pull request merged**